frontend/asset_mapper.rst

```diff
@@ -289,17 +289,25 @@ Preloading and Initializing "app.js"
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In addition to the importmap, the ``{{ importmap() }}`` Twig function also renders
-an `ES module shim`_ and a few other things, like a set of "preloads":
+an `ES module shim`_. You can disable this in your ``config/packages/asset_mapper.yaml``
+configuration file:
```

Comment:
We should probably also document the fact that we can customize the source of the shim, not just disabling it

Reply:
I added it as comment inside the code block. Not very elegant - I have to admit ;-)